doc/sphinx/addendum/micromega.rst

```diff
@@ -145,9 +145,12 @@ and checked to be :math:`-1`.
 
 .. tacn:: lra_Q @ltac_expr
           lra_R @ltac_expr
+          lra_wit_Q @ident @one_term
```

Comment:
It's a trifle odd to have a change log entry announcing something that's for internal use only.  You might mention that it's for internal use only in the changelog.  Or better, add a real description with examples.